src/openrct2/scripting/HookEngine.h

```diff
@@ -37,6 +37,7 @@ namespace OpenRCT2::Scripting
         NETWORK_JOIN,
         NETWORK_LEAVE,
         RIDE_RATINGS_CALCULATE,
+        GAME_ACTION_LOCATION,
```

Comment:
I suggest removing `GAME_` so it is consistent with the other types which are just `ACTION_*`.